File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    //Obtain height from user
    int height;
    do
    {
        height = get_int("Please enter height.");
    }
    while (height < 1 || height > 8);


    //Number of iterations
    for (int i = 0; i < height; i++)
    {
        //Prints empty spaces
        for (int j = height - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        //Prints hashes
        for (int h = 0; h <= i; h++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# asks user for input
while True:
    n = get_int("Height: ")
    if n <= 8 and n >= 1:
        break

for i in range(n):
    for j in range(1, n - i):
        print(" ", end="")
    for m in range(n - i, n + 1):
        print("#", end="")
    print("  ", end="")    
    for h in range(n - i, n + 1):
        print("#", end="")
    print()
 
